src/Frontend/Themes/Bootstrap/Modules/Profiles/Layout/Templates/Settings.html.twig

```diff
@@ -0,0 +1,87 @@
+{% if updateSettingsSuccess %}
+  <div class="alert alert-success" role="alert">{{ 'msg.UpdateSettingsIsSuccess'|trans }}</div>
```

Comment:
tkan mss nuttig zijn om die alerts in een macro te steken? (zodat ge bvb `{{ alert.success('msg.UpdateSettingsIsSuccess') }}` kunt doen?)